### DISCUSSION TITLE
QGTY server implementation

### DISCUSSION THREAD

**Discussion by listepo** [gqty-dev/gqty#2025] [Ideas]
I think we need server implementation with shared types

**Comment by vicary** [comment]
Our friends have been doing server side integrations for gqty, give them a try!
1. [`getcronit/pylon`](https://github.com/getcronit/pylon)
2. [`stepci/garph`](https://github.com/stepci/garph)

While we are focusing on client-side DX, we are also open to ideas on server implementations.

With absolutely no commitment, it may happen in gqty if you have great ideas on server API and no one else bothers doing it.
